src/live.asp.net/AppSettings.cs

```diff
@@ -16,5 +16,7 @@ public class AppSettings
         public string AzureStorageBlobName { get; set; }
 
         public string AzureStorageContainerName { get; set; }
+
+		public string CDNUrl { get; set; }
```

Comment:
maybe some different settings in the editor, I didnt' see any space in my editor. Should I check something specific?